dpnp/dpnp_utils/dpnp_utils_linearalgebra.py

```diff
@@ -331,9 +332,9 @@ def _get_result_shape_vecdot(x1, x2, x1_ndim, x2_ndim):
 
     if x1_ndim == 1 and x2_ndim == 1:
         result_shape = ()
-    elif x1_is_1D:
+    elif x1_ndim == 1:
```

Comment:
It seems `x1_is_1D` and `x2_is_1D` never used now. Do we need to remove `_define_dim_flags(...)` calls above?